File: app/src/main/java/com/studyproject/MainActivity.java
package com.studyproject;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity implements View.OnClickListener {

    private Button bt1;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        initView();
        initListener();
    }

    private void initView() {
        bt1 = (Button) findViewById(R.id.bt1);
    }

    private void initListener() {
        bt1.setOnClickListener(this);
    }

    @Override
    public void onClick(View v) {
        switch (v.getId()){
            case R.id.bt1:
                Intent intent = new Intent(this,SV_Activity.class);
                startActivity(intent);
                break;
        }
    }
}

File: kotilndemo/src/main/java/com/kotilndemo/ui/MainActivity.kt
package com.kotilndemo.ui

import android.app.Activity
import android.os.Bundle
import android.view.View
import android.widget.Toast
import com.kotilndemo.R
import com.kotilndemo.bean.ShopData

class MainActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

    }

    override fun initView() {

    }

    override fun initListener() {

    }

    override fun initData() {

        ShopData.Version
        var dataList = mutableListOf<ShopData>()

    }

    override fun onClick(v: View?) {

    }

fun Activity.showToast(msg:String){
    Toast.makeText(this,msg,Toast.LENGTH_SHORT).show()
}

}
